DiverseSelector/test/test_ClusteringBased.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# The DiverseSelector library provides a set of tools to select molecule
+# subset with maximum molecular diversity.
+#
+# Copyright (C) 2022 The QC-Devs Community
+#
+# This file is part of DiverseSelector.
+#
+# DiverseSelector is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# DiverseSelector is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+
+"""Testing for the dissimilarity-based selection algorithms."""
+
```

Comment:
The file name should be *test_clustering_based.py*. This is a convention not to use capitalized letters in file names.